pkg/addons/data/Service/default/router.yaml

```diff
@@ -20,3 +20,4 @@ spec:
     port: 1936
   selector:
     router: router
+  type: LoadBalancer
```

Comment:
should service.beta.kubernetes.io/azure-load-balancer-mode be specified?

Reply:
tbc